renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "rebaseWhen": "conflicted",
+  "gitAuthor": "Renovate <renovate@oztechan.com>",
+  "commitBody": "Co-authored-by: Mustafa Ozhan <mr.mustafa.ozhan@gmail.com>",
+  "commitMessageAction": "[Oztechan/TraceFit#11] Dependency update",
+  "git-submodules": {
+    "enabled": true,
+    "groupName": "Git Submodules"
+  },
+  "lockFileMaintenance": {
+    "enabled": true,
+    "commitMessageAction": "[Oztechan/TraceFit#11] Lock file maintenance"
+  },
```

Comment:
This configuration is for the lock file of `Gemfile` which will be used later for iOS fastlane builds, the lock file contains the versions of the Gem dependencies, and we need to update them as we update like how we update the gradle dependencies, in order to gain the latest improvements 🙂 

An example PR to visualize what it will look like: https://github.com/Oztechan/CCC/pull/2936/files

Reply:
I see, thank you!